File: Exercise/Easy/11. Check Divisibility/Code/Check Divisibility.ts
function CheckDivisibility(num1 :number, num2 :number) {
    if (num1 % num2 === 0) {
        console.log("Divisible");
    }
    else {
        console.log("Not Divisible");
    }
}

CheckDivisibility(10, 2);
CheckDivisibility(10, 3);

File: Exercise/Easy/11. Check Divisibility/Code/Check Divisibility.java
import java.util.Scanner;

public class CheckDivisibility {
    public static void checkDivisibility(int num1, int num2) {
        if (num1 % num2 == 0) {
            System.out.println("Divisible");
        }
        else {
            System.out.println("Not Divisible");
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        scanner.close();

        checkDivisibility(num1, num2);
    }
}


